examples/cli-version-compare/script#fully-synchronized/o.txt

```diff
@@ -0,0 +1,3 @@
+Reach's Docker images are up-to-date.
```

Comment:
Did you mean to commit the empty [examples/cli-version-compare/script#fully-synchronized/reach](https://github.com/reach-sh/reach-lang/pull/557/files#diff-3bc127310e6b35ed9c9d3c87fdeb49b4cf31e70d20ba4eb615b5ab237d908ab3)?

Reply:
These are intentional, yeah - they pretend there's a newer version of the script available. `README.md` and `run.sh` explain how/why.